x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/160_union_types.yml

```diff
@@ -147,6 +147,114 @@ setup:
           - '{"index": {}}'
           - '{"@timestamp": "2023-10-23T12:15:03.360Z", "client_ip": "172.21.2.162", "event_duration": "3450233", "message": "Connected to 10.1.0.3"}'
 
+  - do:
```

Comment:
Perhaps all these should not be in a union_types test file since it has nothing to do with union-types, but with invalid mapped fields. It is closer to index patterns. Putting it in 50_index_patterns.yml also looks wrong. Perhaps 51_many_indexes.yml?

Reply:
:+1: